modules/investigating-kernel-crashes.adoc

```diff
@@ -2,162 +2,13 @@
 //
 // * support/troubleshooting/troubleshooting-operating-system-issues.adoc
 
-:_content-type: PROCEDURE
+:_content-type: CONCEPT
 [id="investigating-kernel-crashes"]
```

Comment:
Looks like this existing file and the 2 new modules are missing the `_{context}` suffix in their `id`s.